client/a8c-for-agencies/sections/marketplace/hooks/use-product-and-plans.tsx

```diff
@@ -77,6 +77,20 @@ const getDisplayableJetpackProducts = ( filteredProductsAndBundles: APIProductFa
 	} ) as APIProductFamilyProduct[];
 };
 
+const getDisplayableFeaturedProducts = (
+	filteredProductsAndBundles: APIProductFamilyProduct[]
+) => {
+	const featured_product_slugs = [
+		'woocommerce-advanced-notifications',
+		'jetpack-boost',
+		'woocommerce-bulk-stock-management',
+	]; // For now, we hardcode this until we understand how we want pick featured products.
```

Comment:
Is there any reason to mix snake_case and not camelCase for this variable?

Reply:
Also I'd suggest updating this on the backend because this should be the source of truth for the data, and it will be much easier to make any changes later. This can be done as a follow-up, but let's fix the naming for consistency.

Reply:
Out of curiosity, why we picked these products as featured?

Reply:
> Is there any reason to mix snake_case and not camelCase for this variable?
Thanks for catching this. This is now fixed. 

> Also I'd suggest updating this on the backend because this should be the source of truth for the data, and it will be much easier to make any changes later. This can be done as a follow-up, but let's fix the naming for consistency.

I agree this should be handled in the backend. We can determine which endpoint we want to add later on a follow up.


> Out of curiosity, why we picked these products as featured?

I am not sure yet how this was decided. maybe @simonktnga8c @jeffgolenski  and @madebynoam can give context.